AppCenterIdentity/AppCenterIdentityTests/MSIdentityTests.m

```diff
@@ -979,10 +981,33 @@ - (void)testSignOutDoesNothingWhenDisabled {
   [MSIdentity signOut];
 
   // Then
-  XCTAssertEqual([[MSAuthTokenContext sharedInstance] authToken], authToken);
-  XCTAssertEqual([MSMockKeychainUtil stringForKey:kMSIdentityAuthTokenKey], authToken);
-  XCTAssertEqual([self.settingsMock objectForKey:kMSIdentityMSALAccountHomeAccountKey], accountId);
+  XCTAssertEqualObjects([[MSAuthTokenContext sharedInstance] authToken], authToken);
+  XCTAssertEqualObjects([MSMockKeychainUtil stringForKey:kMSIdentityAuthTokenKey], authToken);
+  XCTAssertEqualObjects([self.settingsMock objectForKey:kMSIdentityMSALAccountHomeAccountKey], accountId);
   [identityMock stopMocking];
 }
 
+- (void)testDefaultConfigURL {
+
+  // Then
+  XCTAssertEqualObjects(self.sut.configURL, kMSIdentityDefaultBaseURL);
```

Comment:
can you make that test the same as the next one but without calling setConfigUrl to test the default?